backend/routes/calendarMgmt.js

```diff
@@ -16,9 +16,18 @@ const upload = multer({ dest: 'uploads/' });
 // API Endpoint for Adding New Calendar Events
 router.post('/:user_id/events', authenticateToken, async (req, res) => {
     const {user_id} = req.params;
-    const {title, startAt, endAt, description, location, allDay} = req.body;
+    const {title, startAt, endAt, description, location, allDay, invitees} = req.body;
     try {
-    const newEvent = await prisma.calendarEvent.create({
+        // Retrieve the organizer's email
+        const organizer = await prisma.user.findUnique({
+            where: { id: parseInt(user_id) },
+            select: { email: true }
+        });
+        // Filter out the organizer's email from invitees if present
+        const filteredInvitees = invitees.filter(email => email !== organizer.email);
```

Comment:
you can also gate this at the front end, by not allowing the user to enter their own email in the form ui.